pkg/distro/rhel/rhel10/ami.go

```diff
@@ -157,10 +157,10 @@ func mkEC2SapImgTypeX86_64(osVersion string) *rhel.ImageType {
 // IMAGE CONFIG
 
 // TODO: move these to the EC2 environment
-const (
-	amiKernelOptions        = "console=tty0 console=ttyS0,115200n8 nvme_core.io_timeout=4294967295"
-	amiAarch64KernelOptions = amiKernelOptions + " iommu.strict=0"
-	amiSapKernelOptions     = amiKernelOptions + " processor.max_cstate=1 intel_idle.max_cstate=1"
+var (
```

Comment:
I'll comment on this change once (though it's done in many places), would it make sense to change this to a function that returns `[]string` instead of turning `const` to `var`? I'm always slightly worried about global mutable values.

Reply:
I am too, and I agree.  I'd say I wanted to minimise the diff but tbh it was probably laziness...

Reply:
Done!